### PR TITLE
Users/sbuilta/user requests and maybe rate limits

### PR DESCRIPTION
**Pull request by sBuiltaJr**
This PR incorporates rate limiting and basic word-filter features into the bot.  It also adds a default (AI generated, of course) image; and adds the ability for the bot to handle user-generated prompts.

### CODE REVIEW

File: imageGenSD.py [hunk 1]
```diff
@@ -82,6 +86,24 @@ def GetLoop(self):
...
+       Input  : None.
+
+       Output : None.
+    """
+    result = False;
+    
+    #The default prompt is assumed to not be banned.
+    if banned_words != None and prompt != None:
+        #re-sanatize the word list in case spaces/separators snuck in
+        word_list = (banned_words.replace(" ", "")).split(",")
```
Comment (CrowCrowAI) [inline imageGenSD.py:103]: Is the **replace** basically a strip? or do you want to get rid of spaces between words too?

**Comment by CrowCrowAI** [comment]
Test convo

**Review by CrowCrowAI — CHANGES_REQUESTED** [review]
Test request change

File: QueueMgr.py [hunk 2]
```diff
@@ -11,6 +11,7 @@
 import multiprocessing as mp
 import requests as req
 from urllib.parse import urljoin
+import time
 
 jobs = {}
```
[the remark this answers is not in the document]
Comment (sBuiltaJr) [reply QueueMgr.py:16]: #WeDidItReddit

File: imageGenSD.py (hunk 1 above)
Comment (CrowCrowAI) [reply imageGenSD.py:103]: It fine

Comment (sBuiltaJr) [reply imageGenSD.py:103]: Yeah, it's basically a strip.  I don't think I can remove all spaces and commas in a single strip command.

**Review by CrowCrowAI — APPROVED** [review]

**Pull request merged**